manifests/init.pp

```diff
@@ -126,7 +126,7 @@
     package { 'opsview-agent-removal':
       name   => 'opsview-agent',
       ensure => absent,
-      before => Package[$package_name],
+      before => User[$nagios_user],
```

Comment:
Didn't you say in the daily that the service need to be stopped before removing? On line 148 you have an example for that.

Reply:
Why does this happen before the user is created?

Reply:
Stopping service opsview-agent (legacy init.d script which is sucky) does not work as one might hope. 
I put the before in so that it happens early. At least last time I tried to change a compute node it actually didn't leave an opsview-agent (nrpe process) running.